src/components/organisms/LinkTab/LinkTab.tsx

```diff
@@ -0,0 +1,48 @@
+import { LinkListItem, LINK_TYPE } from "../../atoms/LinkListItem/LinkListItem"
+
+export const LinkTab = () => {
+  const dummyData = [
+    {
+      index: 4,
+      platform: LINK_TYPE["YOUTUBE"],
+      linkTitle: "YOUTUBE",
+      linkDiscription: "",
+      linkDestination: "https://youtube.com/",
+    },
+    {
+      index: 14,
+      platform: LINK_TYPE["TWITTER"],
+      linkTitle: "Twitter",
+      linkDiscription: "",
+      linkDestination: "https://twitter.com/",
+    },
+    {
+      index: 24,
+      platform: LINK_TYPE["INSTAGRAM"],
+      linkTitle: "Instagram",
+      linkDiscription: "",
+      linkDestination: "https://Instagram.com/",
+    },
+    {
+      index: 34,
+      platform: LINK_TYPE["OTHER"],
+      linkTitle: "サイト",
+      linkDiscription: "https://www.google.com/",
+      linkDestination: "https://www.google.com/",
+    },
+  ]
+
+  return (
+    <div className="w-[390px]  gap-y-4 gap-x-3 ">
+      {dummyData.map((v) => (
+        <LinkListItem
+          key={v.index}
+          platform={v.platform}
+          linkTitle={v.linkTitle}
+          linkDiscription={v.linkDiscription}
+          linkDestination={v.linkDestination}
+        />
+      ))}
+    </div>
```

Comment:
- ul liタグを使用する方がセマンティクスで可読性もいいかなと思いましたー！
- map のkeyはindexを以下のようにするといい感じです 👍 dummyDataのindexも不要になります！

```suggestion
    <ul className="w-[390px] gap-y-4 gap-x-3">
      {dummyData.map((v, index) => (
        <li key={index}>
          <LinkListItem
            platform={v.platform}
            linkTitle={v.linkTitle}
            linkDiscription={v.linkDiscription}
            linkDestination={v.linkDestination}
          />
        </li>
      ))}
    </ul>
```

Reply:
> ul liタグを使用する方がセマンティクスで可読性もいいかなと思いましたー！

たしかに！　修正しました！


> map のkeyはindexを以下のようにするといい感じです 👍 dummyDataのindexも不要になります！

すいません！　dummyDataのindexはLinkDBに入っている一意の値を入れるつもりでした。プロパティ名は正しくはindexではなくkeyですね。
アイテムが変動するリストのkeyにindexを指定するのは推奨されないっぽいんで<li key={v.key}>とさせてもらいました！
今回のリストの仕様だと別に<li key=index>でもいいのかな。。。